File: Common/InMotionGIT.Common - Copy/Services/Contracts/Language.vb
Imports System.Runtime.Serialization

Namespace Services.Contracts

    <DataContract()>
    Public Class Language
        Inherits DataType.LookUpValue

        <DataMember()>
        Public Property CulturalCode As String

        <DataMember()>
        Public Property LanguageId As Integer

    End Class

End Namespace

File: Common/InMotionGIT.Common/Services/Contracts/Language.cs
using System.Runtime.Serialization;

namespace InMotionGIT.Common.Services.Contracts
{

    [DataContract()]
    public class Language : DataType.LookUpValue
    {

        [DataMember()]
        public string CulturalCode { get; set; }

        [DataMember()]
        public int LanguageId { get; set; }

    }

}